SUBDOMAIN_TAKEOVER/meta.json

```diff
@@ -0,0 +1,11 @@
+{
+  "risk_rating": "medium",
+  "short_description": "The subdomain is vulnerable to subdomain Takeover",
+  "references": {
+    "Subdomain takeover\n": "https://developer.mozilla.org/en-US/docs/Web/Security/Subdomain_takeovers",
+    "CWE-16": "https://cwe.mitre.org/data/definitions/16.html"
+  },
+  "title": "subdomain takeover",
+  "privacy_issue": false,
+  "security_issue": true
+}
```

Comment:
Please add a recommendation. You can check https://www.honeybadger.io/blog/subdomain-takeover/ for inspiration. The recommendations section is relevant.